docs/getting_started/install.rst

```diff
@@ -58,9 +64,9 @@ Installing Previous Releases
 
 You can also install a specific version (from `jwst 0.17.0` onward):
 
-    | >> conda create -n <env_name> python
+    | >> conda create -n <env_name> python=3.11
     | >> conda activate <env_name>
-    | >> pip install jwst==1.3.3
+    | >> pip install jwst==1.12.5
```

Comment:
Do you also want to update the section below this one, about specific versions before jwst 0.17.0. Or should we remove this section?

Reply:
I'm tempted to remove it, because it's so old and obsolete now.

Reply:
removed in latest commit